src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -146,6 +146,27 @@ enum class SvgParserLengthType
     Other
 };
 
+enum class BBoxFlag
+{
+    None = 0x0,
+    Width = 0x01,
+    Height = 0x02,
+    ViewboxX = 0x04,
+    ViewboxY = 0x08,
+    ViewboxW = 0x1,
+    ViewboxH = 0x2,
+    Viewbox = ViewboxX | ViewboxY | ViewboxW | ViewboxH,
+};
+inline BBoxFlag operator|(BBoxFlag b1, BBoxFlag b2) {
+    return (BBoxFlag)((uint32_t)b1 | (uint32_t)b2);
+}
+inline BBoxFlag operator&(BBoxFlag b1, BBoxFlag b2) {
+    return (BBoxFlag)((uint32_t)b1 & (uint32_t)b2);
+}
+inline BBoxFlag& operator|=(BBoxFlag& b1, BBoxFlag b2) {
+    return (BBoxFlag&)((uint32_t&)b1 |= (uint32_t)b2);
+}
```

Comment:
nice try! I don't know if it's appropriate to declare it here. Haha.. I expect that the readability will be much better if the existing masking codes are handled like this.

Reply:
removed for now -> todo in a separate pr